javascript/packages/eslint-config-nebenan-base/index.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   extends: ['airbnb-base'],
+  parser: "babel-eslint",
```

Comment:
Had to switch to babel-eslint because eslints default parser doesn't support all ECMAScript2020 features. 

Note: Haven't checked if every of our packages which uses eslint-config-nebenan-base uses babel

Reply:
park this and let's wait a bit, it's probably too early to have this feature